### DISCUSSION TITLE
GitHub actions cancelled workflow despite not setting cancel-in-progress

### DISCUSSION THREAD

**Discussion by samkio** [community/community#33948] [Actions]
Hello there!
I have two workflows:
* pr-open.yml : https://github.com/samkio/repo-sandbox/blob/main/.github/workflows/pr-open.yml
* pr-close.yml : https://github.com/samkio/repo-sandbox/blob/main/.github/workflows/pr-close.yml

I have the concurrency set for these two workflows as:
```
concurrency:
  group: pr-${{ github.event.number }}
```

Expectation:
If I close a PR whilst the open PR workflow is still running that the pr-close workflow will queue waiting for the pr-open workflow to finish. The docs say that cancel-in-progress can be set to true to cancel the in progress workflow but as you can see from the files above that is not set.

Reality:
I did see the pr-close workflow queue but half way through executing the pr-open workflow I get the following error:
```
Run pnpm nx run-many --target=build
undefined
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command was killed with SIGTERM (Termination): nx run-many --target=build
Error: Process completed with exit code 1.
```

Here is said workflow run: https://github.com/samkio/repo-sandbox/actions/runs/3113039501/jobs/5047171471
And the queued workflow: https://github.com/samkio/repo-sandbox/actions/runs/3113040053

Am I doing something wrong or is this a bug when using concurrency? I do not expect my pr-open workflow to get a SIGTERM!

**Comment by dclayton-godaddy** [comment]
I have the same issue and I explicitly set `cancel-in-progress: false`.
